.github/workflows/ruby.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+
+name: Ruby
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+    # To automatically get bug fixes and new Ruby versions for ruby/setup-ruby,
+    # change this to (see https://github.com/ruby/setup-ruby#versioning):
+    # uses: ruby/setup-ruby@v1
+      uses: ruby/setup-ruby@ec106b438a1ff6ff109590de34ddc62c540232e0
+      with:
+        ruby-version: 2.6
```

Comment:
At the moment this is using ruby 2.6.6 (see the [build logs here](https://github.com/ruby/setup-ruby)). As per the [documentation](https://github.com/ruby/setup-ruby#supported-version-syntax) I think we can remove this line as it'll try our `.ruby-version` file. One less place to have to update/check these values :)

Reply:
The documentation says "short version like 2.6, automatically using the latest release matching that version (2.6.5)".
Doesn't this then match our .ruby-version file?